pkg/ndctl/ndctl.go

```diff
@@ -66,6 +66,8 @@ func (ctx *Context) CreateNamespace(opts CreateNamespaceOpts) (*Namespace, error
 		for _, r := range bus.ActiveRegions() {
 			if ns, err := r.CreateNamespace(opts); err == nil {
 				return ns, nil
+			} else {
+				return nil, err
```

Comment:
As of i remember, it was intentional, to try with all available regions to serve the creation request. If upper layers are interested in error would call Region.CreateNamespace().

Reply:
As @avalluri pointed out, this changes the semantic.

Otherwise a shorter way to write this if/else would be
`return r.CreateNamespace(opts)`.

But even better is to wrap errors with `errors.Wrap` from https://github.com/pkg/errors because then additional information and a stack trace gets added at each level.

gRPC logging however currently doesn't show the stack trace, so that part isn't useful at the moment.